common/libs/web-workers/scrypt-wrapper.ts

```diff
@@ -0,0 +1,22 @@
+import { IFullWallet, fromPrivateKey } from 'ethereumjs-wallet';
+import { toBuffer } from 'ethereumjs-util';
+const Worker = require('worker-loader!./workers/scrypt-worker.worker.ts');
```

Comment:
We should setup typing for this, as outlined here: https://github.com/webpack-contrib/worker-loader#integrating-with-typescript. Right now it's an implicit `any`.

Reply:
Still typed as any, but
```
import Worker from 'worker-loader!./workers/scrypt-worker.worker.ts';
```
seems to fix it!